app/frontend/styles/support/_all.scss

```diff
@@ -11,4 +11,4 @@
 @import "_checkboxes-scroll";
 @import "_details";
 @import "_sort";
-@import "_syntax-highlighting";
```

Comment:
`_syntax-highlighting.scss` no longer exists in this location, and is no longer used by support anyway (I think).

Reply:
It is — for the API responses!

Reply:
Ah, well then that’s a bug – won’t work as that file doesn’t exist in this folder!

Reply:
Oh, actually, scratch that – I was wrong. Was relying to much on VS Code’s highlighting…